packages/gatsby-source-wordpress/gatsby-browser.js

```diff
@@ -1 +1 @@
-import "./style.css"
```

Comment:
I think this might be a breaking change for users who relied on these styles existing and added their own styles on top. @wardpeet will this change just show up in the v6 version or does this come out in a minor?

Reply:
Oh, just saw your other comment. 👌

Reply:
We won't do any 3.x releases except hotfixes but they are in a different branch